client/state/sites/test/selectors.js

```diff
@@ -54,20 +55,28 @@ function changeVersion( version, operator = 1 ) {
 	const splitVersion = version.split( '.' );
 
 	if ( operator >= 1 ) {
-		splitVersion[ splitVersion.length - 1 ]++;
+		splitVersion[ splitVersion.length - 1 ] ++;
```

Comment:
I fixed space before `++` and `--` in the follow up commit. I need to fix reformat code in my IDE :)